UnityProject/Assets/Scripts/Strings/ReportTemplates.cs

```diff
@@ -26,12 +26,18 @@ public static class ReportTemplates
 			"<color=blue><size=32>Crew Objectives:</size></color>\n\n"+
 			"<size=24>- Subvert the threat.\n\n- Keep the productivity in the station.</size>";
 
-		public static readonly string StationObjective =
+		public static readonly string DeliveryStationObjective =
```

Comment:
since this string is now only relevant to the delivery objective, it could be moved to that scriptable object

Reply:
Good point.